### DISCUSSION TITLE
minZoom as fitview

### DISCUSSION THREAD

**Discussion by adammo94** [xyflow/xyflow#2187] [Q&A]
Hi, I'd like to create a zoom range, where maxZoom is 1, easy, but minZoom would be a fitview zoom level.
Can someone point me toward the solution?

**Comment by bcakmakoglu** [comment]
You could set the minZoom prop with the `onInit` callback.
Something like this:
```jsx
const [minZoom, setMinZoom] = React.useState(0.1);
const onInit = (reactFlowInstance) => {
    reactFlowInstance.fitView({ padding: 0.6 });
    setMinZoom(reactFlowInstance.getZoom());
};
```

[Here's a sandbox example.](https://codesandbox.io/s/sandpack-project-forked-5w3c2f?file=/App.js)
